mlir/include/mlir/Target/LLVMIR/ModuleImport.h

```diff
@@ -355,6 +355,9 @@ class ModuleImport {
   /// and stores a mapping from the struct to the symbol pointing to the
   /// translated operation.
   void processComdat(const llvm::Comdat *comdat);
+  /// Returns a symbol name for a nameless global. MLIR, in contrast to LLVM,
+  /// always requires a symbol name.
+  FlatSymbolRefAttr getOrCreateFakeSymbolName(llvm::GlobalVariable *globalVar);
```

Comment:
```suggestion
  FlatSymbolRefAttr getOrCreateNamelessSymbolName(llvm::GlobalVariable *globalVar);
```
I think I would go with getOrCreateNamelessSymbolName since Fake has a somewhat negative connotation.